src/GeositeFramework/js/App.js

```diff
@@ -100,6 +104,32 @@
             });
         });
     };
+
+    function registerLaunchpadHandlers(launchpadConfigs) {
+        $('a.launchpad-trigger').click(function() {
+            var launchpadId = $(this).data('launchpad-id'),
+                launchpadConfig = getLaunchpadConfig(launchpadConfigs, launchpadId);
+
+            if (!launchpadConfig) {
+                throw new Error("No configuration found for launchpad id " + launchpadId);
```

Comment:
There should be an `app.log` method which will log client side errors like this to the server log.